index.bs

```diff
@@ -294,6 +298,18 @@ At any point during the creation of the {{Socket}} instance, `connect` may throw
       <dd>Initially the same as the `off` option, the connection continues in plain text until the {{startTls()}} method is called</dd>
     </dl>
   </dd>
+  <dt>
+    {{alpn}} member
+  </dt>
+  <dd>
+    The Application-Layer Protocol Negotiation list to send. If the server agrees with one of the protocols specified in this list, it will return the matching protocol in the {{info}} property. May be specified if and only if {{secureTransport}} is `on` or `starttls`.
```

Comment:
Would this be a single `string` (e.g. `'h1, h2'`)  or an Array of strings (e.g. `['h1', 'h2']`). My preference would be for the latter.

Reply:
And rather than `May be specified if and only if {{secureTransport}}...` might I suggest that we say only that these would be ignored if `{{secureTransport}}` is anything other than `on` or `starttls`.? Likewise with `sni` below.

Reply:
> Would this be a single string (e.g. 'h1, h2') or an Array of strings (e.g. ['h1', 'h2']). My preference would be for the latter.

Good point, I also prefer the latter (we do this in Deno currently and it works), plus the space is potentially valid as ALPN is just a "bag of bytes". It should also be specified that the ALPN protocols specified must be Latin-1 strings.

> And rather than May be specified if and only if {{secureTransport}}... might I suggest that we say only that these would be ignored if {{secureTransport}} is anything other than on or starttls.? Likewise with sni below.

I'm OK either way!

Reply:
@mmastrac ... would you like to go ahead and make these additional edits or would you like me to?